lib/commonAPI/coreapi/ext/platform/android/src/com/rho/webview/WebViewSingleton.java

```diff
@@ -445,6 +445,90 @@ private void readRhoelementsConfig(IRhoConfig config) {
         		mConfig.set(WebViewConfig.SETTING_SPLASHSCREEN_DURATION, splashscreendurationValue);
         	}
         }
+        if(config.isExist("enablefunctionkey_f1")){
+        	String enablefuncKeyF1Value = config.getString("enablefunctionkey_f1");
```

Comment:
Make changes in this file
https://github.com/rhomobile/Motorola-Extensions/blob/master/extensions/rhoelementsext/ext/rhoelementsext/platform/android/src/com/rhoelementsext/RhoElementsExt.java#L323